### PR TITLE
[fix] MuPDF Android font dir patch update

### PR DESCRIPTION
**Pull request by Frenzie**
Cf. <https://github.com/koreader/koreader/issues/5347#issuecomment-529480686>.

### CODE REVIEW

File: thirdparty/mupdf/external_fonts.patch [hunk 1]
```diff
@@ -93,6 +94,22 @@ index f6951ba2..b1fdd981 100644
 +	return filename;
 +}
 +
++char *
++get_font_file_android(char *name)
++{
++	char *fontdir;
```
Comment (NiLuJe) [inline thirdparty/mupdf/external_fonts.patch]:
```C
	const char fontdir[] = "/system/fonts";
	size_t len = sizeof(fontdir) + strlen(name) + 1;	// sizeof on a string literal includes the NUL
	char *filename = malloc(len);
	if (filename == NULL) {
		return NULL;
	}
```

(Fairly minor, this works ;)).

Comment (Frenzie) [reply thirdparty/mupdf/external_fonts.patch]: I'm pretty sure I tested with what I thought of as a stress test (EPUB) but I guess that just used the Times (or whatever) that's bundled with MuPDF. In any case I didn't have any HTML dictionaries installed on the emulator.

Comment (Frenzie) [reply thirdparty/mupdf/external_fonts.patch]: You're fine with the basic principle besides this nit? :-P

Comment (NiLuJe) [reply thirdparty/mupdf/external_fonts.patch]: Yep!

File: thirdparty/mupdf/external_fonts.patch [hunk 2]
```diff
@@ -68,7 +68,7 @@ index f6951ba2..b1fdd981 100644
  /*
  	Base 14 PDF fonts from URW.
  	Noto fonts from Google.
-@@ -367,3 +371,185 @@ fz_lookup_noto_emoji_font(fz_context *ctx, int *size)
+@@ -367,3 +371,188 @@ fz_lookup_noto_emoji_font(fz_context *ctx, int *size)
```
Comment (Frenzie) [inline thirdparty/mupdf/external_fonts.patch]: @NiLuJe Apparently I thought your suggestion took off more lines than it did? :-P Oh well, luckily there's the CI.

**Pull request merged**